{"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"decapsulationKeyInconsistent","kind":"identifier"}],"platforms":["macOS"]}]}],"metadata":{"title":"Exception.decapsulationKeyInconsistent","externalID":"s:10SwiftKyber9ExceptionO28decapsulationKeyInconsistentyA2CmF","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"decapsulationKeyInconsistent"}],"modules":[{"name":"SwiftKyber"}],"symbolKind":"case","roleHeading":"Case"},"variants":[{"paths":["\/documentation\/swiftkyber\/exception\/decapsulationkeyinconsistent"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","sections":[],"hierarchy":{"paths":[["doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/decapsulationKeyInconsistent"},"schemaVersion":{"patch":0,"major":0,"minor":3},"abstract":[{"type":"text","text":"Inconsistent decapsulation key data"}],"references":{"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception/decapsulationKeyInconsistent":{"role":"symbol","type":"topic","title":"Exception.decapsulationKeyInconsistent","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception\/decapsulationKeyInconsistent","abstract":[{"text":"Inconsistent decapsulation key data","type":"text"}],"url":"\/documentation\/swiftkyber\/exception\/decapsulationkeyinconsistent","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"decapsulationKeyInconsistent","kind":"identifier"}]},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber/Exception":{"navigatorTitle":[{"text":"Exception","kind":"identifier"}],"identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber\/Exception","role":"symbol","title":"Exception","type":"topic","url":"\/documentation\/swiftkyber\/exception","kind":"symbol","abstract":[{"text":"The Kyber exceptions","type":"text"}],"fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"Exception","kind":"identifier"}]},"doc://swiftkyber.SwiftKyber/documentation/SwiftKyber":{"type":"topic","role":"collection","kind":"symbol","url":"\/documentation\/swiftkyber","title":"SwiftKyber","identifier":"doc:\/\/swiftkyber.SwiftKyber\/documentation\/SwiftKyber","abstract":[{"type":"text","text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard"}]}}}